src/rust/plugin-sdk/generator-sdk/src/test_utils/test_ctx.rs

```diff
@@ -71,25 +71,28 @@ impl GeneratorTestContextInternals {
             Duration::from_millis(50),
         );
 
-        let service_name = server.service_name();
-
         let server_handle = tokio::task::spawn(server.serve());
 
-        HealthcheckClient::wait_until_healthy(
-            endpoint.clone(),
-            service_name,
+        let client_config = ClientConfiguration::new(
+            address,
+            Duration::from_millis(500),
+            Duration::from_millis(500),
+            2,
+            Duration::from_millis(1),
+            Duration::from_millis(100),
+            Duration::from_millis(10),
+            20,
+            Duration::from_millis(10),
+            Duration::from_millis(250),
+        );
```

Comment:
This is a constructor. It's here because none of the fields in this struct are optional, so a builder pattern isn't applicable. If, as you suggest, we made a constructor like this:

```rust
fn new(SomeStruct) -> Self { ... }
```

we'd still have to specify a constructor for `SomeStruct`.. unless you want to make all the fields on `SomeStruct` public and mutable? I think that's a pretty unfavorable tradeoff for what amounts to an aesthetic issue.

As you note, this constructor generally won't be used, except in tests. These objects are constructed by deserializing something given by a `figment::providers::*`. Currently we're using `figment::providers::Env`.